finquant/portfolio.py

```diff
@@ -1055,6 +1142,7 @@ def build_portfolio(**kwargs):
         "start_date",
         "end_date",
         "data_api",
+        "market_index",
```

Comment:
add market_index to the comment above as well: # 1. pf_allocation, names, start_date, end_date, data_api, market_index